File: zddl.js
// 引入AJAX库（如jQuery或axios）
// 请确保在使用之前已经引入了相关的库

// 目标网站的登录URL
var loginUrl = 'http://36.189.241.20:9956/web';

// 获取用户名和密码的输入框
var usernameInput = document.getElementById('web-auth-user');
var passwordInput = document.getElementById('web-auth-password');

// 填充用户名和密码
usernameInput.value = 'g17881101711';
passwordInput.value = '123123';

// 获取登录按钮
var loginButton = document.getElementById('web-auth-connect');

// 模拟点击登录按钮
loginButton.click();

// 延迟执行断开连接和重新连接的操作
setTimeout(function() {
  // 执行断开连接的操作
  var disconnectButton = document.getElementById('web-auth-disconnect');
  disconnectButton.click();

  setTimeout(function() {
loginButton.click()
    // 执行重新连接的操作
    // ...
  }, 2000); // 延迟 2 秒后执行重新连接的操作
}, 5000); // 延迟 5 秒后执行断开连接的操作

File: zddl.py
import requests

# 在这里填写你的用户名和密码
username = 'g17881101711'
password = '123123'

# 登录页面的 URL
login_url = 'http://36.189.241.20:9956/web'

# 创建一个会话
session = requests.Session()

# 发送登录请求
login_data = {
    'username': username,
    'password': password
}
response = session.post(login_url, data=login_data)

# 检查登录是否成功
if response.status_code == 200:
    print('登录成功')
else:
    print('登录失败')
